.github/CODEOWNERS

```diff
@@ -138,6 +138,7 @@
 /core/templates/pages/contributor-dashboard-page/ @oppia/contributor-experience-team
 /core/templates/pages/contributor-dashboard-admin-page/ @oppia/contributor-experience-team
 /scripts/populate_sample_contributor_data*.py @oppia/contributor-experience-team
+/core/templates/services/html-length.service.ts @oppia/contributor-experience-team
```

Comment:
Should the contributor experience team also own the associated spec file?

Reply:
Done, PTAL